rpc/client/types.go

```diff
@@ -0,0 +1,11 @@
+package client
+
+// ABCIQueryOptions can be used to provide options for ABCIQuery call other
+// than the DefaultABCIQueryOptions.
+type ABCIQueryOptions struct {
+	Height  uint64
+	Trusted bool
+}
+
+// DefaultABCIQueryOptions are latest height (0) and prove equal to true.
```

Comment:
update comment - "trusted equal to false (which will result in a proof being returned)"